rlwe/elements.go

```diff
@@ -98,6 +98,13 @@ func NewCiphertextNTT(params Parameters, degree, level int) *Ciphertext {
 	return el
 }
 
+// NewCiphertextRandom generates a new uniformly distributed Ciphertext of degree, level and scale.
+func NewCiphertextRandom(prng utils.PRNG, params Parameters, degree, level int) (ciphertext *Ciphertext) {
+	ciphertext = NewCiphertext(params, degree, level)
+	PopulateElementRandom(prng, params, ciphertext)
```

Comment:
This method creates a new Ring from the passed params which is no longer needed.

Reply:
Fixed